### DISCUSSION TITLE
Totally not important typo in dllmain.c

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#541] [General]
I've been struggling away unsuccessfully so far to implement [#431](https://github.com/valinet/ExplorerPatcher/discussions/431) myself and maybe put in a PR when I noticed this totally unimportant typo in dllmain.c 
"BOOL ShowLegacyClockExpierience(HWND hWnd)"

**Comment by valinet** [comment]
Yeah, oops… I will correct it. It’s tricky to spot with the naked eye too. Thx.

**Reply by valinet** [reply]
Done, I will push eventually soon.
